### ISSUE TITLE
instead of severeal slips ports, use the port as a random string as a prefix to all keys

### ISSUE THREAD

**Issue by AlyaGomaa** [stratosphereips/StratosphereLinuxIPS#207]
Created by Alya Gomaa via monday.com integration. 🎉

**Comment by A-atmos** [comment]
Would you please elaborate the issue, I wanted to work on it.

**Comment by A-atmos** [comment]
hey @AlyaGomaa , could we just not use multiple dbs for different instance of slips in the same redis server, instead of adding `random string` as prefixes to identify the instance. This would resolve in using a single redis server for all instance of slips in a single port!

**Comment by AlyaGomaa** [comment]
yes this is the goal, we want to have 1 single redis server for all instances

**Comment by A-atmos** [comment]
hey @AlyaGomaa , I was working on this and have a prototype which is passing the database tests, but is not giving out the alert logs. I could make a PR and may be you could give me a lead on it.
For now I have:
1. Added a prefix to each keys, so as to maintain the keys for different instance of slips
2. Changed core slips files to maintain and work by using prefix instead of port.
3. Modified database tests to use the prefix.
4. Introduced an argument which can let use a custom prefix instead of a generated uuid4.

Also I've confusions on if we continue using different port for kalipso and web interface, or change them to use different prefixes accordingly.
